src/lib/comms/Makefile

```diff
@@ -16,8 +16,20 @@ obj-networking-$(COAP) += \
     sol-coap.o
 
 obj-networking-$(OIC) += \
+    sol-oic-cbor.o \
     sol-oic-client.o \
-    sol-oic-server.o
+    sol-oic-server.o \
+    $(TINYCBOR_SRC_PATH)/cborencoder.o \
+    $(TINYCBOR_SRC_PATH)/cborerrorstrings.o \
+    $(TINYCBOR_SRC_PATH)/cborparser.o \
+    $(TINYCBOR_SRC_PATH)/cborpretty.o
+
+obj-networking-$(OIC)-extra-cflags := \
+    -I$(TINYCBOR_SRC_PATH) \
+    -Wno-cpp \
+    -Wno-declaration-after-statement \
+    -Wno-float-equal \
+    -Wno-undef
```

Comment:
are these because of cbor? if so could we add a TODO FIXME so we remember to remove those once they are changed in that library? (most look like valid errors that should get warnings).